src/card.css

```diff
@@ -0,0 +1,90 @@
+* {
+  box-sizing: border-box;
+}
+
+.card-box {
+  width: 200px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+
+  font-family: "Roboto", Helvetica, sans-serif;
+  color: #060b35;
+  font-weight: 400;
+  font-size: 10px;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+}
+
+.card__img-imac {
```

Comment:
delete .card__img-imac cos what if card will be about huawei and no need in img in div in this case